diff --git a/node_modules/@payloadcms/db-postgres/dist/connect.js b/node_modules/@payloadcms/db-postgres/dist/connect.js
index bf4beed..68fb378 100644
--- a/node_modules/@payloadcms/db-postgres/dist/connect.js
+++ b/node_modules/@payloadcms/db-postgres/dist/connect.js
@@ -11,11 +11,11 @@ const connectWithReconnect = async function({ adapter, pool, reconnect = false }
         } catch (ignore) {
             setTimeout(()=>{
                 adapter.payload.logger.info('Reconnecting to postgres');
-                void connectWithReconnect({
+                connectWithReconnect({
                     adapter,
                     pool,
                     reconnect: true
-                });
+                }).catch(()=>{});
             }, 1000);
         }
     }
@@ -25,11 +25,11 @@ const connectWithReconnect = async function({ adapter, pool, reconnect = false }
     result.prependListener('error', (err)=>{
         try {
             if (err.code === 'ECONNRESET') {
-                void connectWithReconnect({
+                connectWithReconnect({
                     adapter,
                     pool,
                     reconnect: true
-                });
+                }).catch(()=>{});
             }
         } catch (ignore) {
         // swallow error
@@ -62,10 +62,10 @@ export const connect = async function connect(options = {
                     connectionString
                 };
                 const pool = new this.pg.Pool(options);
-                void connectWithReconnect({
+                connectWithReconnect({
                     adapter: this,
                     pool
-                });
+                }).catch(()=>{});
                 return drizzle({
                     client: pool,
                     logger,
